dwave/system/composites/linear_ancilla.py

```diff
@@ -0,0 +1,214 @@
+import dimod
+
+
+__all__ = ["LinearAncillaComposite"]
+
+
+class LinearAncillaComposite(dimod.ComposedSampler, dimod.Structured):
+    """Implements linear fields as ancilla qubits polarized with strong flux biases.
+
+    Linear field `h_i` of qubit `i` is implemented through a coupling `J_{ij}` between
+    the qubit and a neighbouring qubit `j` that is fully polarized with a large flux bias.
+
+    Args:
+    child_sampler (:class:`dimod.Sampler`):
+        A dimod sampler, such as a :obj:`DWaveSampler`, that has flux bias controls.
+
+    """
+
+    def __init__(
+        self,
+        child_sampler: dimod.Sampler,
+    ):
+        self.children = [child_sampler]
+        self.parameters = child_sampler.parameters.copy()
+        self.properties = dict(child_properties=child_sampler.properties.copy())
+        self.nodelist = child_sampler.nodelist
+        self.edgelist = child_sampler.edgelist
+
+    def nodelist(self):
+        pass  # overwritten by init
+
+    def edgelist(self):
+        pass  # overwritten by init
+
+    children = None  # overwritten by init
+    """list [child_sampler]: List containing the structured sampler."""
+
+    parameters = None  # overwritten by init
+    """dict[str, list]: Parameters in the form of a dict.
+
+    For an instantiated composed sampler, keys are the keyword parameters
+    accepted by the child sampler and parameters added by the composite.
+    """
+
+    properties = None  # overwritten by init
+    """dict: Properties in the form of a dict.
+
+    Contains the properties of the child sampler.
+    """
+
+    @nonblocking_sample_method
+    def sample(
+        self,
+        bqm: dimod.BinaryQuadraticModel,
+        *,
+        h_tolerance: numbers.Number = 0,
+        default_flux_bias_range: Sequence[float] = [-0.005, 0.005],
+        **parameters,
+    ):
+        """Sample from the provided binary quadratic model.
+
+        .. note::
+            This composite does not suport the auto_scale parameter. BQM Scaling
+            can be done with :class:`dimod.ScaleComposite`.
+
+        Args:
+            bqm (:obj:`~dimod.BinaryQuadraticModel`):
+                Binary quadratic model to be sampled from.
+
+            h_tolerance (:class:`numbers.Number`):
+                Magnitude of the linear bias can be left on the qubit. Assumed to be positive. Defaults to zero.
+
+            default_flux_bias_range (:class:`typing.Sequence`):
+                Flux bias range safely accepted by the QPU, the larger the better.
+
+            **parameters:
+                Parameters for the sampling method, specified by the child
+                sampler.
+
+        Returns:
+            :obj:`~dimod.SampleSet`
+
+        """
+        if h_tolerance < 0:
+            raise ValueError("h_tolerance needs to be positive or zero")
+
+        child = self.child
+        qpu_properties = innermost_child_properties(child)
+        g_target = child.to_networkx_graph()
```

Comment:
why not `target_graph` and `source_graph`?

Reply:
Good point. I think I have a tendency to math typing as in $G_{\rm target}$ so that the object type is upfront and the adjective later. I've implemented your suggestion since that's how it appears in Ocean